server/config/local.json-dist

```diff
@@ -27,6 +27,11 @@
   "metrics": {
     "sample_rate": 1
   },
+  "sentry": {
+    "sample_rate": 1,
+    "endpoint": "https://sentry.prod.mozaws.net/api/31/store/",
+    "api_key": "8e7aaaa38fdb46e58ec0b3de61373492"
```

Comment:
This gives devs the ability to send errors to our sentry endpoint, but requires current devs to update their local.json file. Is that the gist? How can we look up errors produced from our local dev environment on Sentry?